docs/Reference/Console/Features/ocr.md

```diff
@@ -0,0 +1,87 @@
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_HTTPSBOOT_BIOS.png" alt="Figure 1: Enable HTTPS Boot in BIOS">
+    </figure>
+3. Set up an HTTPS server to host the ISO. For faster booting, run it on the same network as the device.
+    - This guide references using [simple-webserver](https://github.com/jsastriawan/simple-webserver) for the HTTPS server setup. 
+    - In this example, the HTTPS server is running on the same machine as the Console: https://192.168.88.250:5443/ubuntu.iso.
+    - The hosted ISO is a full Ubuntu LTS image served via simple-webserver.
+
+### Triggering OCR via Console
+
+1. Connect to the Intel AMT device using Console over TLS.
+
+2. Enable `HTTPS Network boot` feature in `General AMT Info` Section.
+   
+    !!! note "Note - HTTPS Network Boot supported?"
+
+        If the HTTPS Network Boot checkbox is greyed out, HTTPS Network Boot is not supported on this platform.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_Enable_HTTPS_BOOT.png" alt="Figure 2: Enable HTTPS Network Boot">
+    </figure>
+
+3. Upload the Root Certificate of the HTTPS server hosting the ISO via the `Add New` certificates option.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_ADD_TRUSTEDROOTCERT.png" alt="Figure 3: Add Root Certificate of HTTPS Server">
+    </figure>
+
+4. Click on the three-dot menu and select **Reset to HTTPS Boot (OCR)**.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_Reset_to_HTTPS_Boot.png" alt="Figure 4: Reset to HTTPS Boot (OCR)">
+    </figure>
+
+5. Enter the ISO URL (e.g., https://192.168.88.250:5443/ubuntu.iso). Optionally, enable `Enforce Secure Boot` to boot only a secure `.iso` file.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_HTTPSBOOT_URL.png" alt="Figure 5: URL to the .iso hosted on HTTPS Server">
+    </figure>
+
+6. Click `OK` to start the recovery process. The device will restart and boot from the ISO.
+
+7. Optionally, Connect to KVM and verify that the device loads the ISO.
+    
+    <figure class="figure-image">
+      <img src="..\..\..\..\assets\images\OCR_HTTPSBoot_Recovery_Start.png" alt="Figure 7: View KVM screen while the ISO boots">
+    </figure>
+ 
+### Troubleshooting and Known issues
+
+- *Wired Connection Required* – OCR in Console works only when the device is connected via a wired network.
+
+- *Check ISO URL* – Ensure the HTTPS Server ISO URL is accessible to the device.
+
+- *Secure Boot Issues* – If Secure Boot is enabled, the BIOS must trust the certificate used to sign the ISO.
+
+- *OEM Support Varies* – Not all OEMs support this feature.
+
+- *TLS Connection Required* – Console must be connected to AMT over TLS for this feature to work.
+
+- *KVM Keyboard Issue* – If KVM is connected before initiating OCR via HTTPS Boot, the keyboard may not work when entering the URL.
```

Comment:
I would make this  a known issue with a WARNING Admonition next to the step about using KVM.